cli/internal/config/config.go

```diff
@@ -194,8 +198,7 @@ func ParseAndValidate(args []string, ui cli.Ui, turboVersion string) (c *Config,
 		Output: output,
 	})
 
-	maxRemoteFailCount := 3
-	apiClient := client.NewClient(partialConfig.ApiUrl, logger, turboVersion, partialConfig.TeamId, partialConfig.TeamSlug, uint64(maxRemoteFailCount), usePreflight)
+	maxRemoteFailCount := uint64(3)
```

Comment:
Kind of a weird place to set a constant.

Reply:
It both was and is a weird place. I'm not 100% sure we'd even make this one configurable. That being said, I want to punt on it because I think we'll eventually want to tackle config for `ApiClient` as a whole. That one is probably especially relevant for `viper`, since it pulls from a couple config files.